### DISCUSSION TITLE
html2image 2.0.3 no longer allows for size changing

### DISCUSSION THREAD

**Discussion by eboehning** [vgalin/html2image#112] [Q&A] [answered]
Hi all,

I've been using html2image2.0.1 for months and recently noticed the screenshot function stopped working. After updating to 2.0.3, I see that I am able to screenshot again, but not able to alter the size of the screenshot. Anyone else having this problem? Any ideas to fix it? Doesn't look like anything with the size has changed, but still doesn't work

**Comment by vgalin** [comment, marked as answer]
see https://github.com/vgalin/html2image/issues/103
